docs/source/manim_directive.py

```diff
@@ -123,6 +137,13 @@ class ManimDirective(Directive):
     final_argument_whitespace = True
 
     def run(self):
+        if "skip-manim" in self.state.document.settings.env.app.builder.tags.tags:
+            node = skip_manim_node()
+            self.state.nested_parse(
+                StringList(self.content[0]), self.content_offset, node
+            )
+            return [node]
```

Comment:
Is this return value used somewhere?
I can not find it.

Reply:
This is used by Sphinx at some later point.